src/library/scala/math/Equiv.scala

```diff
@@ -185,13 +189,13 @@ object Equiv extends LowPriorityEquiv {
     *                     relation for `NaN` (it is not reflexive), there are two
     *                     equivalences defined for `Double`: `StrictEquiv`, which
     *                     is reflexive, and `IeeeEquiv`, which is consistent
-    *                     with IEEE spec and doubleing point operations defined in
+    *                     with IEEE spec and floating point operations defined in
```

Comment:
🤣

Reply:
oops

Reply:
please tell me this isn't also in `Ordering`...

Reply:
it's not :-)